src/app/common/model/confirmation-modal-data.ts

```diff
@@ -0,0 +1,6 @@
+export interface IConfirmationModalData{
```

Comment:
The convention in the project is to name files containing types with `.types.ts` in the end. So this one can be named for example `confirmation-modal.types.ts` or `confirmation-modal-config.types.ts` if you want to have only one interface in it but I would go maybe for the first option. We often have more than one interface in the file.